content-src/styles/_theme.scss

```diff
@@ -77,7 +77,7 @@ body {
 }
 
 // Dark theme
-.dark-theme {
+body[lwt-newtab-brighttext] {
```

Comment:
We have a sass linter that only returns warnings (won't fail on travis) and it's complaining about this line. In this case I wouldn't recommend fixing it, instead you can just suppress the warnings by adding this at the end.
```
// sass-lint:disable-line force-attribute-nesting, no-qualifying-elements
```
You can run the linter by running `npm run lint` (notice that we have quite a few but we're trying not to add more).